algorithm_services/app.py

```diff
@@ -1,20 +1,53 @@
 import json
+import markdown
+import re
+import os.path
+import jinja2
 
 from flask import Flask
 from flask import render_template
 from flask import Response
+from flask import Markup
 
 from algorithm_services.algorithms.fizzbuzz import fizzbuzz
 from algorithm_services.algorithms.clock_angle import clock_angle
 
 
+def get_readme_algorithms_doc(readme_data):
+    found = re.search(
+        '### Available algorithms(.+?)### Tests',
+        readme_data,
+        re.DOTALL
+    )
+    if found:
+        algorithms_data = """### Basic usage
+        - Syntax: <server domain>/<algorithm (lower case)>/<inputs>
+        - Example: localhost:8888/fizzbuzz/20
+        """
+        algorithms_data += '\n### Available Algorithms\n'
+        algorithms_data += found.group(1)
+    return algorithms_data
+
+
 def create_app(name, config):
     app = Flask(name)
     app.config.update(config)
 
+    app_loader = jinja2.FileSystemLoader(
+                os.path.abspath('./algorithm_services/templates') # noqa
```

Comment:
Use the config in your favor; set a new config attribute to store templates folder path.